packages/vimebac/vimebac.0.0.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+version: "0.0.0"
+maintainer: "Sebastien Mondet <seb@mondet.org>"
+authors: ["Sebastien Mondet <seb@mondet.org>"]
+homepage: "https://gitlab.com/smondet/vimebac"
+bug-reports: "https://gitlab.com/smondet/vimebac/issues"
+dev-repo: "https://gitlab.com/smondet/vimebac.git"
+license: "ISC"
+build: [
+  ["jbuilder" "build" "-p" "vimebac" "-j" jobs]
+]
+depends: [
+  "nonstd"
+  "sosa"
+  "cmdliner"
+  "misuja"
+  "wall"
+  "tsdl"
+  "rresult"
+  "tgls"
+  "jbuilder" {build & >= "1.0+beta20"}
+  "ocamlfind" {build}
```

Comment:
Also do you need `ocamlfind` there ? I can't find any reference to either `ocamlfind` or `Findlib` in your source code